### PR TITLE
Functional referral

### PR DESCRIPTION
**Pull request by BrandonC123**
Task: 
first lets change "Enter Wallet REFFERAL LINK" with "REFERRER" & hide send button then

1-) user should be able to generate / copy referral link of themselves  (by using copy button in second textfield)

2-) the ones clicked the referral link will have their referrer text field filled  (the first text field)

3-) make the reward dynamic usd equivalent of 0.5 eth

### CODE REVIEW

**Comment by BrandonC123** [comment]
Hiding the button was causing some issues with styling so refactored it a bit to be more dynamic. Current look:
image

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@emotion/server": "^11.4.0",
     "@emotion/styled": "^11.3.0",
     "@ethersproject/contracts": "^5.5.0",
+    "@firebase/storage": "^0.9.14",
```
Comment (tanerdurmaz) [inline package.json:24]: we dont need this

File: pages/index.tsx [hunk 2]
```diff
@@ -83,157 +83,182 @@ const Home: NextPage = () => {
       hash: "bafybeidn5ubtxclqpr55l5gocwstop5moqccgoakhclqxx3uiegdu5fofi"
     }
 
+  ]);
```
Comment (tanerdurmaz) [inline pages/index.tsx]: this update shouldnt change index.tsx, if its only indentation issue latest precommit script should fix it you can pull it from main branch

File: styles/globals.css [hunk 3]
```diff
@@ -5,71 +5,80 @@
...
+    top: 50%;
+    left: 8%;
+    transform: translateY(-50%);
+    font-size: 5rem;
+    font-family: "Poppins";
+    margin: 0;
+    line-height: 1em;
+}
+
+.referral-row {
```
Comment (tanerdurmaz) [inline styles/globals.css]: lets dont add these in global.css 
we have tailwind now

**Review by tanerdurmaz — CHANGES_REQUESTED** [review]
also lets change default values in input text boxes. 

referrer will have "enter wallet address ex. 0x4ba..."
referral link will have "connect your wallet" if wallet is not connected

**Comment by BrandonC123** [comment]
Seems I messed up a couple lines of code while merging index.tsx initially, should be good now. Here's the updated referral with and without wallet connected:
conne
w ref

**Review by tanerdurmaz — APPROVED** [review]

**Pull request merged**